webapp/src/main/java/com/box/l10n/mojito/service/tm/TMService.java

```diff
@@ -709,7 +710,11 @@ public String generateLocalized(
 
         driver.addStep(new RawDocumentToFilterEventsStep());
         driver.addStep(new CheckForDoNotTranslateStep());
-        driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
+        if (bcp47Tag.equals("en-x-psaccent")) {
+            driver.addStep(new PseudoLocalizeStep());
+        } else {
+            driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
```

Comment:
I want to have separate CLI command for pseudoloc.  But I don't want to create a separate function in TMService which is very similar to generateLocalized function.  Instead of doing checking `en-x-psaccent".equals(bcp47Tag)`, we can pass in a flag to do pseudoloc.